alidock/helpers/init.sh.j2

```diff
@@ -16,10 +16,13 @@ function _alidock_ps1() {
   [[ $RV != 0 ]] && echo -n "$RV|"
 }
 export -f _alidock_ps1
-export PS1='`_alidock_ps1`[alidock] \w \$> '
+export PS1='`_alidock_ps1`[{{dockName}}] \w \$> '
```

Comment:
If you specify --name <container> you will get [container] in the shell prompt. You can effectively run/manage different containers at the same time (but beware, keep homes separated as well).